src/@[scope]/[name].md.js

```diff
@@ -95,13 +95,20 @@ import semverCompare from "npm:semver/functions/compare";
 const downloads = JSON.parse(data__downloads.textContent, reviver);
 const versions = JSON.parse(data__versions.textContent, reviver);
 const commits = JSON.parse(data__commits.textContent, reviver);
+const issues = JSON.parse(data__issues.textContent, reviver);
 const lastYear = new Date("${formatIso(lastYear)}");
 const start = new Date("${formatIso(start)}");
 const today = new Date("${formatIso(today)}");
 const domain = [start, today];
 
 function reviver(key, value) {
-  return typeof value === "string" && /(^|_)(date|time)$/.test(key) ? new Date(value) : value;
+  return typeof value === "string" && /(^|_)(date|time|at)$/.test(key) ? new Date(value) : value;
+}
+
+function idle() {
```

Comment:
Could this become an integral part of framework somehow? Like, an option of resize, and even an option of fenced code blocks?